ui/src/components/table/TableHeader.jsx

```diff
@@ -71,13 +76,28 @@ function TableHeader({
         );
     };
 
+    const getStatusCount = () => {
+        const enabledRowCount = allFilteredData.filter((item) => !item.disabled).length;
+
+        if (totalElement <= pageSize) {
+            return '';
+        }
+
+        return (
+            <StatusCountWrapper>
```

Comment:
could you use `<Typography as="span">` instead? This text looks inconsistent because Splunk styles are not applied to font

Reply:
probably worth to adjust totalElement as well